.gitignore

```diff
@@ -1,3 +1,4 @@
 /vendor
 /composer.lock
 /phpunit.xml
+/.phpunit.result.cache
```

Comment:
don;t need this

Reply:
why? it's boring. Let's add it!

Reply:
(but please, could you sort all entries?)

Reply:
It was only added because they upgraded phpunit, then they downgraded it, and it's no longer needed.